packages/reporting-server/setup.py

```diff
@@ -0,0 +1,40 @@
+from setuptools import find_packages, setup
+
+package_name = "reporting_server"
+
+setup(
+    name=package_name,
+    description="RMF reporting server",
+    version="0.0.0",
+    packages=find_packages(exclude=["tests"]),
+    author="Matias Bavera",
+    author_email="matiasbavera@gmail.com",
+    keywords=["RMF", "reporting"],
+    classifiers=[
+        "Intended Audience :: Developers",
+        "License :: OSI Approved :: Apache Software License",
+        "Programming Language :: Python",
+        "Topic :: Software Development",
+    ],
+    install_requires=[
+        "fastapi~=0.63.0",
+        "aiofiles~=0.6.0",
```

Comment:
this is needed only when serving static files, does it actually use it?

Reply:
At least no for now, I'll remove it.